src/net.cpp

```diff
@@ -1797,6 +1809,9 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
             return;
 
         // Add seed nodes if DNS seeds are all down (an infrastructure attack?).
+        // Note that we only do this if we started with an empty peers.dat,
+        // (in which case we will query DNS seeds immediately) *and* the DNS
+        // seeds have not returned any results.
```

Comment:
I don't think calling those peers "fixed seeds" matches with DNS seeds meaning. Contrary to DNS seeds open as `fOneShot=true`, we open regular connections to them, and not querying a `ADDR` and then disconnecting ?

Reply:
I agree it's a bit confusing that we call both of them seeds, but the problem is not with this PR. This PR just adapts to the existing naming.
Are you suggesting renaming fixed seeds to hardcoded nodes or something?
That alone would require a separate PR.

Reply:
Yes I'm suggesting renaming them with a different names to mark difference of mechanism in a separate PR.

Reply:
I'm going to merge this now. @ariard please open a followup for this (and if you want the other nits).